Issues tutorial for building from source

I was following the tutorial (https://github.com/kubernetes-incubator/cri-o/blob/master/tutorial.md) step by step (on a debian machine) but encountered the following issues:

* The `Get crictl` step relies on git, but is only installed in the next step

* Tutorial refers to install Go 1.8.5, but later version is required:

```bash
# github.com/kubernetes-incubator/cri-o/vendor/k8s.io/kubernetes/pkg/kubelet/server/streaming
vendor/k8s.io/kubernetes/pkg/kubelet/server/streaming/server.go:245: s.server.ServeTLS undefined (type *http.Server has no field or method ServeTLS)
Makefile:99: recipe for target 'bin/crio' failed
make: *** [bin/crio] Error 2
```
Updating go to the latest version (1.10.3) fixed the problem

* Missing libudev during make, needed to do `apt-get install libudev-dev`

* Missing  go-md2man during make, needed to do `apt-get install go-md2man`

* Missing  add-apt-repository, had to install `apt-get install software-properties-common`

--- a/tutorials/setup.md
+++ b/tutorials/setup.md
@@ -102,7 +102,11 @@ apt-get update -qq && apt-get install -y \
   libselinux1-dev \
   pkg-config \
   go-md2man \
-  cri-o-runc
+  cri-o-runc \
+  libudev-dev \
+  software-properties-common \
+  gcc \
+  make
 ```
 
 **Caveats and Notes:**

